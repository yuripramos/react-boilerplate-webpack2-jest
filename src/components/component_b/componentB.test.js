import React from 'react';
import { shallow } from 'enzyme';

import ComponentB from './componentB.js';

const props = {
	lastname: "State 2",
};

describe('rendering tests', () => {
	it('rendering correctly', () => {
		const component = shallow(<ComponentB {...props} />);
		expect(component).toMatchSnapshot();
	});
});
