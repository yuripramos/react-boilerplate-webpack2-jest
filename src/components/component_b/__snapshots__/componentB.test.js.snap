// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering tests renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-12"
  >
    <p>
       
      undefined don't forget the component B
       
    </p>
    <p>
       
      undefined don't forget the component B
       
    </p>
    <p>
       
      undefined don't forget the component B
       
    </p>
    <p>
       
      undefined don't forget the component B
       
    </p>
    <p>
       
      undefined don't forget the component B
       
    </p>
    <p>
       
      undefined don't forget the component B
       
    </p>
    <p>
       
      undefined don't forget the component B
       
    </p>
    <p>
       
      undefined don't forget the component B
       
    </p>
    <p>
       
      undefined don't forget the component B
       
    </p>
  </div>
</div>
`;
